tests/apitests/python/library/configurations.py

```diff
@@ -2,25 +2,77 @@
 
 import base
 import swagger_client
+from swagger_client.rest import ApiException
+
+def set_configurations(client, expect_status_code = 200, expect_response_body = None, **config):
+    conf = swagger_client.Configurations()
+    if "project_creation_restriction" in config:
+        conf.project_creation_restriction = config.get("project_creation_restriction")
+    if "token_expiration" in config:
+        conf.token_expiration = config.get("token_expiration")
+    HasError = False
+    try:
+        _, status_code, _ = client.configurations_put_with_http_info(conf)
+    except ApiException as e:
+        HasError = True
+        if e.status == expect_status_code:
+            if expect_response_body is not None and e.body.strip() != expect_response_body.strip():
+                raise Exception(r"Set configuration response body is not as expected {} actual status is {}.".format(expect_response_body.strip(), e.body.strip()))
```

Comment:
nit: rephrase as follows:

`Set configuration response body is not as we expected. Expected {}, while actual status is {}`.

Reply:
Thanks, corrected.